### DISCUSSION TITLE
Costs when transferring a domain from a third party into Vercel

### DISCUSSION THREAD

**Discussion by ChristianIvicevic** [vercel/community#776] [Help] [answered]
I have been considering to transfer most of my domains into Vercel, but I cannot find any information on the costs of this process. Most of my domains are running out somwhere in November and will automatically extend unless I cancel the contracts. As such I am looking for some clarifications.

1. Suppose I'd transfer today, I guess I'd be paying Vercel for a year from today and thus have "wasted" the money I have spent for the few months until my contracts would have naturally run out, right?
2. When transfering a domain to Vercel are there any fees associated with this process on Vercel's side?
3. How can I check the price of the domains beforehand? Do same TLDs share the same price regardless of actual domain name?

**Comment by swarnava** [comment, marked as answer]
there are no extra charges that come with a transfer. For most [domain extensions](https://www.namecheap.com/domains/new-tlds/explore/), you will just be charged with a regular transfer fee. This fee includes regular domain renewal fee.
 
Note that the above is a general procedure, and it may vary from TLD to TLD, so if you need to find out more, you should reach out to the current registrar support team for more details.
